include/dpp/auditlog.h

```diff
@@ -33,152 +33,160 @@ namespace dpp {
-	ae_automod_rule_delete		=	142,
+	aut_automod_rule_delete		=	142,
 	/// Auto moderation block message
-	ae_automod_block_message	=	143,
+	aut_automod_block_message	=	143,
 };
 
 /**
  * @brief Defines audit log changes
  */
 struct DPP_EXPORT audit_change {
-	/// Optional: Serialised new value of the key
+	/// Optional: Serialised new value of the change, e.g. for nicknames, the new nickname
 	std::string	new_value;
-	/// Optional: Serialised old value of the key
+	/// Optional: Serialised old value of the change, e.g. for nicknames, the old nickname
 	std::string	old_value;
-	/// name of audit log change key
+	/**
+	 * The property name that was changed, e.g. `nick` for nickname changes
+	 * @note For dpp::audit_type::aut_appcommand_permission_update updates the key is the id of the user, channel, role, or a permission constant that was updated instead of an actual property name
+	 */
 	std::string	key;
 };
 
 /**
  * @brief Extra information for an audit log entry
  */
 struct DPP_EXPORT audit_extra {
-	std::string 	delete_member_days;	//!< number of days after which inactive members were kicked
+	std::string delete_member_days;	//!< number of days after which inactive members were kicked
 	std::string	members_removed;	//!< number of members removed by the prune
 	snowflake	channel_id;		//!< channel in which the entities were targeted
 	snowflake	message_id;		//!< id of the message that was targeted
 	std::string	count;			//!< number of entities that were targeted
 	snowflake	id;			//!< id of the overwritten entity
 	std::string	type;			//!< type of overwritten entity - "0" for "role" or "1" for "member"
-	std::string 	role_name;		//!< name of the role if type is "0" (not present if type is "1")
+	std::string role_name;		//!< name of the role if type is "0" (not present if type is "1")
+	snowflake	application_id;	//!< ID of the app whose permissions were targeted
 };
 
 /**
  * @brief An individual audit log entry
  */
 struct DPP_EXPORT audit_entry {
 	snowflake			id;		//!< id of the entry
-	snowflake			target_id;	//!< id of the affected entity (webhook, user, role, etc.) (may be empty)
+	/**
+	 * ID of the affected entity (webhook, user, role, etc.) (may be empty)
+	 * @note For dpp::audit_type::aut_appcommand_permission_update updates, it's the command ID or the app ID
+	 */
+	snowflake			target_id;
 	std::vector<audit_change>	changes;	//!< Optional: changes made to the target_id
-	snowflake			user_id;	//!< the user who made the changes (may be empty)
-	audit_type			event;		//!< type of action that occurred
-	std::optional<audit_extra>	options;	//!< Optional: additional info for certain action types
-	std::string			reason;		//!< Optional: the reason for the change (0-512 characters)	
+	snowflake			user_id;	//!< the user or app that made the changes (may be empty)
+	audit_type			type;		//!< type of action that occurred
+	std::optional<audit_extra>	extra;	//!< Optional: additional info for certain action types
```

Comment:
renamed options, discordjs and python lib also named it like that. the options attribute was broke anyways, so a breaking change wouldn't affect anyone.